docs/flatcar-linux/packet.md

```diff
@@ -239,6 +239,35 @@ supernova-helium-worker-1  Ready   node               10m  v1.14.1
 
 Learn about [maintenance](../topics/maintenance.md).
 
+## ARM Support and Hybrid Clusters
+
+Lokomotive and Flatcar support the Packet arm64 server types `c1.large.arm` and `c2.large.arm`.
+They can be used for both worker and controller nodes.
+Besides specifying them in `controller_type`/`type` you need to configure some additional variables
+in the respective controller/worker module:
+
+```
+os_arch = "arm64"
+os_channel = "alpha"
+ipxe_script_url = "https://raw.githubusercontent.com/kinvolk/flatcar-ipxe-scripts/arm64-usr/packet.ipxe"
+```
+
+The channel variable is needed as long as the content of the iPXE script refers to `alpha.release…`.
+Adjust it to the named channel there or remove it once the script uses `stable.release…`.
```

Comment:
Is it when the URL has that string or can you please paste an example?

Reply:
Tried to make it more clear by saying that it's the value for `base-url`.